docs/guides/recipe.rst

```diff
@@ -110,9 +110,170 @@ the Recipe.
+
+.. code-block:: yaml
+
+  flow:
+  - type: DAG
+    annotations: {}
+    inputs:
+    - type: DAGFileInput
+      annotations: {}
+      name: idf
+      description: Input energyplus idf file.
+      default: null
+      alias: []
+      required: true
+      spec: null
+      extensions:
+      - .idf
+    - type: DAGFileInput
+      annotations: {}
+      name: weather
+      description: Input wea weather file.
+      default: null
+      alias: []
+      required: true
+      spec: null
+      extensions:
+      - .epw
+    outputs:
+    - type: DAGFolderOutput
+      annotations: {}
+      name: outputs
+      description: output folder
+      from:
+        type: FolderReference
+        annotations: {}
+        path: ep-results
+      alias: []
+      required: true
+    name: f634208a54c93a7f08a38336187e7e17b437bd929e4f56d3b16b7b9510b496cb/ep-run
+    fail_fast: true
+    tasks:
+    - type: DAGTask
+      annotations: {}
+      name: run-ep-simulation
+      template: 31686b3b4a263a7cffa4048d70eed36e7b294a0a0cb8318ae9f296fa400f9501/run-simulation
+      needs: []
+      arguments:
+      - type: TaskPathArgument
+        annotations: {}
+        name: weather
+        from:
+          type: InputFileReference
+          annotations: {}
+          variable: weather
+        sub_path: null
+      - type: TaskPathArgument
+        annotations: {}
+        name: idf
+        from:
+          type: InputFileReference
+          annotations: {}
+          variable: idf
+        sub_path: null
+      loop: null
+      sub_folder: null
+      returns:
+      - type: TaskPathReturn
+        annotations: {}
+        name: outputs
+        description: null
+        path: ep-results
+        required: true
+  - type: DAG
+    annotations: {}
+    inputs:
+    - type: DAGArrayInput
+      annotations: {}
+      name: cities
+      description: A list of names that matches the name of the weather file. Without
+        the epw (e.g. chicago). A weather file name chicago.epw should exist in weather_files
+        folder.
+      default: []
+      alias: []
+      required: true
+      spec: null
+      items_type: String
+    - type: DAGFileInput
+      annotations: {}
+      name: idf
+      description: Path to input idf file.
+      default: null
+      alias: []
+      required: true
+      spec: null
+      extensions:
+      - idf
+    - type: DAGFolderInput
+      annotations: {}
+      name: weather-files
+      description: A folder with several epw files. The simulation will be executed
+        for each weather file in parallel.
+      default: null
+      alias: []
+      required: true
+      spec: null
+    outputs: []
+    name: f634208a54c93a7f08a38336187e7e17b437bd929e4f56d3b16b7b9510b496cb/main
+    fail_fast: true
+    tasks:
+    - type: DAGTask
+      annotations: {}
+      name: energy-simulation-parallel-run
+      template: f634208a54c93a7f08a38336187e7e17b437bd929e4f56d3b16b7b9510b496cb/ep-run
```

Comment:
Hi @tymokvo, I know this is a draft PR but I thought I should point this out and save you some time. This YAML is from a baked-recipe. In a non-baked `Recipe` the template is referred to by name instead of hash and Queenbee resolves those dependencies when "baking" them. That keeps the original recipes human-readable and fixes the dependencies in the baked version. Here is the recipe as a folder - not being baked.

[parametric-energy-run.zip](https://github.com/pollination/queenbee/files/6015260/parametric-energy-run.zip)

You can also generate it yourself if you want. I didn't think of a case where someone wants to generate the non-baked version of the recipe but it is possible to do. You can use `--dry-run` option of `push` command to get a non-baked version of the recipe. It prints out the path to where the recipe has been exported to.

`pollination dsl push pollination-parametric-energy-run --dry-run`

Reply:
Ah, thank you!